src/SprykerEco/Zed/Heidelpay/Business/Encrypter/AesEncrypter.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Business\Encrypter;
+
+use SprykerEco\Zed\Heidelpay\HeidelpayConfig;
+
+class AesEncrypter implements EncrypterInterface
+{
+
```

Comment:
Did you run the CS?

Reply:
I'm going to run CS for dev branch after merge, because the current state is too bad.

Reply:
Unit tests would be very good here, especially if it is about encryption and security functionality :)